Makefile

```diff
@@ -16,29 +23,96 @@ osrm:
+
+cypress:
+	@npm run e2e
+# For now, just change here the `cypress/e2e/xxx/file.cy.js` to run a specific test
+cypress-only:
+	@clear && cypress run --browser chrome --headless --no-runner-ui --spec cypress/e2e/local-commerce/@admin/update_price.cy.js
+cypress-only-until-fail:
+	@while clear && cypress run --browser chrome --headless --no-runner-ui --spec cypress/e2e/local-commerce/@admin/update_price.cy.js; do :; done
+cypress-open:
+	@cypress open
+cypress-install:
+	@docker compose exec -e APP_ENV=test -e SYMFONY_ENV=test -e NODE_ENV=test webpack npm install -g cypress@14.3.2 @cypress/webpack-preprocessor@6.0.4
+	@npm install -g cypress@14.3.2 @cypress/webpack-preprocessor@6.0.4
 
 jest:
-	@docker compose exec -e SYMFONY_ENV=test -e NODE_ENV=test webpack npm run jest
+	@docker compose exec -e APP_ENV=test -e SYMFONY_ENV=test -e NODE_ENV=test webpack npm run jest
 
-migrations-diff:
-	@docker compose exec php php bin/console doctrine:migrations:diff --no-interaction
+# Just an alias
+migrations: migrations-migrate
 
 migrations-migrate:
 	@docker compose exec php php bin/console doctrine:migrations:migrate
 	@docker compose exec php php bin/console doctrine:schema:update --env=test --force --no-interaction --complete
 
+migrations-diff:
+	@docker compose exec php php bin/console doctrine:migrations:diff --no-interaction
+
 email-preview:
 	@docker compose exec php php bin/console coopcycle:email:preview > /tmp/coopcycle_email_layout.html && open /tmp/coopcycle_email_layout.html
 
 enable-xdebug:
 	@docker compose exec php /usr/local/bin/enable-xdebug
 	@docker compose restart php nginx
 
+start:
+	@clear && docker compose up --remove-orphans
+
+# Once everything is restarted, you need to run in another terminal: `make setup`
+start-fresh: fresh-db fresh
+
 fresh:
-	@docker compose down
+	@clear && docker compose down
+	@docker compose up --build --remove-orphans
+
+fresh-db:
+	@docker compose up -d postgres
+	@docker compose exec -T postgres dropdb -U postgres coopcycle
+	@docker compose exec -T postgres dropdb -U postgres coopcycle_test
+
+# This one solves weird file permissions issues when
+# browsing the `test` env at http://localhost:9080/
+perms:
+	@docker compose exec php sh -c "chown -R www-data:www-data web/ var/ && chmod 777 web/ var/ && chmod 666 web/build/entrypoints.json"
+
+lint:
+	@docker compose exec php php vendor/bin/phpstan analyse -v
+
+test: phpunit jest behat cypress
+
+testdata1:
+	@docker compose exec php bin/console coopcycle:fixtures:load -f cypress/fixtures/high_volume_instance.yml --env test
+testdata2:
+	@docker compose exec php bin/console coopcycle:fixtures:load -f cypress/fixtures/dispatch.yml --env test
+testdata3:
+	@docker compose exec php bin/console coopcycle:fixtures:load -f cypress/fixtures/package_delivery_orders.yml --env test
+testdata4:
+	@docker compose exec php bin/console coopcycle:fixtures:load -f cypress/fixtures/restaurant.yml --env test
```

Comment:
Can those have more explicit names?

Reply:
Also, wondering what is the usage of this? Each Cypress scenario will load the needed fixtures, no?

Reply:
Done! Changed them to be more verbose:
```
testdata-dispatch:
testdata-restaurant:
testdata-package-delivery-orders:
testdata-high-volume-instance:
```
The usage is mainly because we often use the `test` local env (like at `http://localhost:9080/` from `web` and `192.168.0.xx:9080` for `app`) because the fixtures in there are handy to test our changes in the mobile app (for example, the @vladimir-8's `high_volume_instance.yml` one)
If we use our `dev` local env, then `make demodata` does the trick for that env.